content/events/2021-online-unconference/_index.md

```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
   before, during, and after the event
   
  
+## Share the event
+
+Do you want to share the event with your colleagues/friends? Here are a few texts that you can reuse, feel also free to modify them as you like
+
+### Short teaser
+#NordicRSE invites everyone interested in "Research Software Engineering" activities to join & shape the agenda of the NordicRSE unconference (lightweight get-together) on June 29-30. See [https://nordic-rse.org/events/2021-online-unconference/](https://nordic-rse.org/events/2021-online-unconference/) for more details #RSEng @EOSC_Nordic @coderefine @SciCompAalto
```

Comment:
Thanks for adding these suggestions. Small comments: I think we should remove the twitter tags and @s, especially the first #NordicRSE might make it into a heading in markdown. Furthermore, NordicRSE should be written Nordic-RSE.

Reply:
good point! updated